package.json

```diff
@@ -41,12 +41,18 @@
   },
   "devDependencies": {
     "babel-cli": "6.26.0",
+    "babel-core": "^6.26.3",
     "babel-jest": "^23.4.2",
+    "babel-loader": "^8.0.4",
     "babel-plugin-rewire": "1.1.0",
     "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-polyfill": "^6.26.0",
     "babel-preset-babili": "0.1.4",
     "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
```

Comment:
We may not require all these new packages, but there were some unsupported es6 features.